test/powershell/Modules/Microsoft.PowerShell.Diagnostics/Export-Counter.Tests.ps1

```diff
@@ -0,0 +1,216 @@
+<############################################################################################
+ # File: Export-Counter.Tests.ps1
+ # Provides Pester tests for the Export-Counter cmdlet.
+ ############################################################################################>
+Describe "Tests for Export-Counter cmdlet" -Tags "CI" {
+
+    BeforeAll {
+        $cmdletName = "Export-Counter"
+
+        . "$PSScriptRoot/CounterTestsCommon.ps1"
```

Comment:
Please add the helper functions in the same file. You can add them before the describe block.